### DISCUSSION TITLE
Clone SQL Login

### DISCUSSION THREAD

**Discussion by danaxs69** [dataplat/dbatools#7639] [Ideas] [answered]
Hi, trying to find a command for clone SQL login. But i cant find it, is it possible to fix a command for that?

br
Danny

**Comment by niphlod** [comment, marked as answer]
clone as what ? doesn't Copy-DbaLogin suffice ?

**Reply by danaxs69** [reply]
Clone Login

source: inst1
dest: inst1
SQL login existing: Login1 
SQL login new: Login2 (with the same user mapping/server roles/sucurables as Login1)

Witch command could i use?

**Reply by niphlod** [reply]
see Example 3 of https://docs.dbatools.io/New-DbaLogin

**Reply by niphlod** [reply]
also, Example 6 of https://docs.dbatools.io/Copy-DbaLogin should do exactly what you need. New-DbaLogin just copies the login without copying permissions

**Reply by danaxs69** [reply]
Thanks a lot, works perfect
